### PR TITLE
merge

### PR DESCRIPTION
**Pull request by Burlesque1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Then the result will be saved at `./config/{car ordinal}.json`
 4. You could modify the log level in `logger.py` for console and file handlers.
 5. Feel free to modify any logic to fit your style.
 
+
```
Comment (Juice-XIJ) [inline README.md:35]: why new line?

File: forza.py [hunk 2]
```diff
@@ -96,23 +96,23 @@ def analyze(self, performance_profile: bool=True):
...
-            if performance_profile:
-                fig, ax = plt.subplots(2, 2)
+            # if performance_profile:
+            #     fig, ax = plt.subplots(2, 2)
 
-                # # gear vs ratio at 0, 0
-                helper.plot_gear_ratio(self, ax, 0, 0)
+            #     # # gear vs ratio at 0, 0
+            #     helper.plot_gear_ratio(self, ax, 0, 0)
+            #     # torque vs rpm at 0, 1
```
Comment (Juice-XIJ) [inline forza.py:104]: Don't include test behaviors, like comment code in the PR. If you want to delete them, please show the reasons.

File: gui.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
+import sys
+
+from PyQt5.QtWidgets import QApplication, QLabel, QWidget, QPushButton,QVBoxLayout, QHBoxLayout, QMainWindow
+from PyQt5.QtGui import QIcon
+from PyQt5.QtCore import *
+
+
+def foo():
```
Comment (Juice-XIJ) [inline gui.py:8]: Test code?

File: gui.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
+import sys
```
Comment (Juice-XIJ) [inline gui.py:1]: Please describe usage examples, including screenshots and feature behaviors. In mean the while, you have to update the readme.md and readme.zhcn.md as well for this change

File: logger.py [hunk 5]
```diff
@@ -15,6 +16,7 @@ def __init__(self):
         console_handler = logging.StreamHandler(sys.stdout)
         console_handler.setLevel(logging.INFO)
 
+
```
Comment (Juice-XIJ) [inline logger.py:19]: why new line?

File: main.py [hunk 6]
```diff
@@ -1,20 +1,20 @@
+from logger import logger
```
Comment (Juice-XIJ) [inline main.py:1]: why re-order these imports? The system path appended should be ahead of local class import.

File: main.py [hunk 7]
```diff
@@ -23,38 +23,46 @@ def on_press(key):
         key: key
     """
     pressed = keyboard_helper.get_key_name(key)
+    if pressed == '`':
```
Comment (Juice-XIJ) [inline main.py:26]: function already have.

File: settings.py [hunk 8]
```diff
@@ -0,0 +1,79 @@
+import os
```
Comment (Juice-XIJ) [inline settings.py:1]: where do we import this file? It's duplicated with constants.

**Comment by Burlesque1** [comment]
no need to merge it right now, everything is sill on the fly. For the best practice plz grant me the permission to create a new branch

**Comment by Juice-XIJ** [comment]
> no need to merge it right now, everything is sill on the fly. For the best practice plz grant me the permission to create a new branch

Got you. Since your purpose is not for merging, I will close the PR. Please don't submit PR if you don't want to merge or be reviewed.

**Pull request closed without merging**